src/routes/+page.server.js

```diff
@@ -18,9 +18,19 @@ export async function load() {
 				}
 				imageCardAltText
 			}
-		}	  
-		
+			agendas {
+				cardImage {
+					url
+				}
+				date
+				event
+				price
+				address
+				buttonText
+			}
+		}
```

Comment:
De load function is een beetje onoverzichtelijk. Als je de query in een apart script bestand zet, en dan hier importeert, wordt het wat leesbaarder..